vnext/e2e/__tests__/TestInputTest.ts

```diff
@@ -0,0 +1,41 @@
+import { webDriver2, HomePage, TextInputTestPage, AnyPage, clickAndWaitForHomePage } from '../testlib/index';
+import { waitForPage } from 'selenium-appium'
+
+jest.setTimeout(30000);
+
+beforeAll(() => {
+  return webDriver2.start();
+});
+
+afterAll(() => {
+  return webDriver2.stop();
+})
+
+describe("TestInputTestPage", () => {
```

Comment:
>describe("TestInputTestPage", [](start = 0, length = 29)

What is this doing?  I'm curious about how a test makes the appropriate RN test page load.